packages/ui/.storybook/main.js

```diff
@@ -22,11 +22,9 @@ module.exports = {
         '../src/stories/**/*.stories.@(js|jsx|ts|tsx)',
     ],
     addons: [
-        'storybook-addon-apollo-client',
```

Comment:
I removed `storybook-addon-apollo-client` and `storybook-addon-next-router` because those weren't needed for `packages/ui`.